python/lsst/ts/tcpip/base_client_or_server.py

```diff
@@ -332,6 +333,32 @@ async def read_into(self, struct: ctypes.Structure) -> None:
             await self._close_client()
             raise
 
+    async def read_str(self) -> str:
+        """Read data ending in `TERMINATOR`.
+
+        Before returning the data, it is decoded into an UTF-8 string.
+
+        Returns
+        -------
+        str:
+            Data decoded into a string.
```

Comment:
Please document the exceptions that this method can raise.

Reply:
Done, see my previous comment.